build/frappe-worker/Dockerfile

```diff
@@ -73,9 +73,12 @@ RUN chown -R frappe:frappe /home/frappe
 COPY build/frappe-worker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 RUN ln -s /usr/local/bin/docker-entrypoint.sh / # backwards compat
 
-USER frappe
 WORKDIR /home/frappe/frappe-bench
 
+RUN chown -R frappe:frappe /home/frappe/frappe-bench
```

Comment:
This like does same for the parent directory.

https://github.com/frappe/frappe_docker/blob/ccc3fc27073b1c6ccd983745d22ef3dc07308aea/build/frappe-worker/Dockerfile#L70

I think this line needs to be moved to line 78 instead of the line you added?